gulpfile.js

```diff
@@ -1063,10 +1072,16 @@ function compileJs(srcDir, srcFilename, destDir, options) {
             .pipe($$.rename(destFilename))
             .pipe(lazywrite())
             .on('end', function() {
-              appendToCompiledFile(srcFilename, destDir + '/' + destFilename);
+              appendToCompiledFile(srcFilename,
+                  path.join(destDir, destFilename));
             }))
         .then(() => {
```

Comment:
Cannot wait until we move to Node 8+ as a requirement so these can become async/await.

Reply:
@kristoferbaxter We currently require node 6+ because it is the lowest Active LTS version.  This will change in a month, when node 8+ becomes the lowest Active LTS version (see https://github.com/nodejs/Release), so I support making Node 8+ a requirement soon.

cc @jridgewell for comment.

Reply:
I think I lost the maintainance LTS debate, so moving to the latest active LTS is fine.

Reply:
PR sent for the bump up to node 8. #13848.

Reply:
PR landed. Node 8 is now the minimum required version for AMP development.